addons/escoria-core/game/core-scripts/esc/commands/change_scene.gd

```diff
@@ -47,35 +47,50 @@ func validate(arguments: Array) -> bool:
 # Run the command
 func run(command_params: Array) -> int:
 	escoria.logger.info(
-		"Changing scene to %s (disable_automatic_transition = %s, run_events = %s)" % [
-		command_params[0],
-		command_params[1],
-		command_params[2]
+		"Changing scene to %s (enable_automatic_transition = %s, run_events = %s)" % [
+		command_params[0],	# scene file
+		command_params[1],	# enable_automatic_transition
+		command_params[2]	# run_events
 	])
 	
+	# Clear the event queue to remove other events (there could be duplicate
+	# events in there so we avoid running these multiple times)
+	escoria.event_manager.clear_event_queue()
+	
+	var exited_previous_room = false
+	
+	if command_params[1] \
+	and escoria.event_manager._running_event.name in ["exit_scene", "debug"]:
```

Comment:
Indent this two times deeper than line 62 to comply with GDScript code guidelines